oqsprov/oqs_prov.h

```diff
@@ -189,6 +192,25 @@ struct oqsx_key_st {
 
 typedef struct oqsx_key_st OQSX_KEY;
 
+// composite signature
+struct SignatureModel {
+    ASN1_BIT_STRING *sig1;
+    ASN1_BIT_STRING *sig2;
+};
+
+typedef struct SignatureModel CompositeSignature;
+
+char *get_oqsname_fromtls(char *tlsname);
+char *get_oqsname(int nid);
+char *get_cmpname(int nid, int index);
+int get_oqsalg_idx(int nid);
+int get_composite_idx(int idx);
+
+/* Workaround for not functioning EC PARAM initialization
+ * TBD, check https://github.com/openssl/openssl/issues/16989
```

Comment:
Is this still accurate (issue seems closed)?

Reply:
I followed the same pattern as the hybrid does, and the setECParams() is still used for the classical key recreate.
I didn't check if the setECParams() workaround is still accurate.
https://github.com/open-quantum-safe/oqs-provider/blob/main/oqsprov/oqsprov_keys.c#L533C1-L535C18

Reply:
ACK, so I didn't. So out of scope for this PR.